README.md

```diff
@@ -33,6 +33,98 @@ import { ChowlyClient } from '@goparrot/chowly-api-client';
         .catch(error => console.log(error));
 })();
 ```
+```javascript
+const ChowlyClient = require('./dist/index');
+
+const Customer = {
```

Comment:
no need to provide such details, use something like `const Customer = { /* your customer data */  };`

Reply:
add examples in `.ts`

Reply:
better to use `customer` instead of `Customer`